component/custom/custom.go

```diff
@@ -0,0 +1,67 @@
+package custom
+
+import (
+	"os"
+	"path/filepath"
+	"plugin"
+	"strings"
+
+	"github.com/Azure/azure-storage-fuse/v2/common/log"
+	"github.com/Azure/azure-storage-fuse/v2/exported"
+	"github.com/Azure/azure-storage-fuse/v2/internal"
+)
+
+func init() {
```

Comment:
UT can be written to explicitly call the init method of this module and give a directory with some dummy .so files and validate one shall load successfully, one shall fail and all those combinations like .so does not exists, path is invalid etc.

Reply:
As init() cannot be invoked explicitly used a helper method instead to test. Also dummy logic won't work with .so files. so had to validate via sample custom components.